app/utils/report/templates/build.txt

```diff
@@ -57,38 +57,6 @@ Section mismatches summary:
 {%- if error_details %}
 {{ "{:=^80}".format("") }}
 
```

Comment:
These lines need to be removed too, they are part of the `error_details` if-block.